### ISSUE TITLE
Elementor from nuget does not work

### ISSUE THREAD

**Issue by chcnetconsulting** [peachpiecompiler/peachpie#962]
```
PHP5018	/wp-content/plugins/elementor/includes/base/widget-base.php (135,12):	Deprecated: Method 'Elementor\Widget_Base::_register_skins' has been deprecated. 3.1.0
PHP5018	/wp-content/plugins/elementor/includes/base/controls-stack.php (1951,11):	Deprecated: Method 'Elementor\Controls_Stack::_register_controls' has been deprecated. 3.1.0 Use `Controls_Stack::register_controls()` instead
PHP5018	/wp-content/plugins/elementor/includes/base/element-base.php (673,12):	Deprecated: Method 'Elementor\Element_Base::_add_render_attributes' has been deprecated. 3.1.0
PHP5018	/wp-content/plugins/elementor/includes/base/controls-stack.php (921,28):	Deprecated: Method 'Elementor\Controls_Stack::_get_initial_config' has been deprecated. 2.9.0 use `get_initial_config()` instead
PHP5018	/wp-content/plugins/elementor/includes/base/controls-stack.php (2096,12):	Deprecated: Method 'Elementor\Controls_Stack::_init' has been deprecated. 2.9.0 use `init()` instead
PHP5018	/wp-content/plugins/elementor/includes/base/controls-stack.php (1631,11):	Deprecated: Method 'Elementor\Controls_Stack::_content_template' has been deprecated. 2.9.0 use `content_template()` instead
PHP5018	/wp-content/plugins/elementor/core/dynamic-tags/base-tag.php (177,11):	Deprecated: Method 'Elementor\Controls_Stack::_register_controls' has been deprecated. 3.1.0 Use `Controls_Stack::register_controls()` instead
PHP5018	/wp-content/plugins/elementor/core/common/modules/connect/apps/base-app.php (539,22):	Deprecated: Method 'Elementor\Core\Common\Modules\Connect\Apps\Base_App::request' has been deprecated. Please use `http_request` method instead of this method.
PHP5018	/wp-content/plugins/elementor/core/utils/import-export/wp-import.php (623,47):	Deprecated: Method 'wp_slash_strings_only' has been deprecated. 5.6.0 Use wp_slash()
PHP5018	/wp-content/plugins/elementor/core/common/modules/connect/apps/base-app.php (593,11):	Deprecated: Method 'Elementor\Core\Common\Modules\Connect\Apps\Base_App::request' has been deprecated. Please use `http_request` method instead of this method.
PHP5018	/wp-content/plugins/elementor/core/utils/import-export/wp-import.php (861,79):	Deprecated: Method 'wp_slash_strings_only' has been deprecated. 5.6.0 Use wp_slash()
PHP5018	/wp-content/plugins/elementor/core/utils/import-export/wp-import.php (895,50):	Deprecated: Method 'wp_slash_strings_only' has been deprecated. 5.6.0 Use wp_slash()
PHP5018	/wp-content/plugins/elementor/core/common/modules/connect/apps/library.php (50,30):	Deprecated: Method 'Elementor\Core\Common\Modules\Connect\Apps\Base_App::request' has been deprecated. Please use `http_request` method instead of this method.
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (662,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (677,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (682,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (724,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (734,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (776,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (805,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (831,11):	Call to undefined method: Upgrades::notice()
PHP5009	/wp-content/plugins/elementor/core/upgrade/upgrades.php (841,11):	Call to undefined method: Upgrades::notice()
PHP5018	/wp-content/plugins/elementor/core/common/modules/connect/apps/base-app.php (123,22):	Deprecated: Method 'Elementor\Core\Common\Modules\Connect\Apps\Base_App::request' has been deprecated. Please use `http_request` method instead of this method.
PHP5018	/wp-content/plugins/elementor/core/common/modules/connect/apps/base-app.php (177,22):	Deprecated: Method 'Elementor\Core\Common\Modules\Connect\Apps\Base_App::request' has been deprecated. Please use `http_request` method instead of this method.
Dismiss this notice.
Plugin could not be activated because it triggered a fatal error.
```

**Comment by jakubmisek** [comment]
Is it installed from a nuget package? because nuget packages should not trigger any kind of compilation nor `PHPxxxx` error codes.

**Comment by chcnetconsulting** [comment]
Hi,

Wordpress itself is working, but plugins not. How to reproduce:

install peachpie-wordpress from git, add the plugin elementor, rebuild and run with dotnet run -p app

and this is the result: Plugin could not be activated because of a fatal error:

**Comment by jakubmisek** [comment]
thank you, will try that

**Comment by jakubmisek** [comment]
Ok, we have a bit mismatch in versioning.

You need `elementor` version `3.2.2-preview7`. The other versions in the feed were for testing purposes and are not compatible with the WordPress 5.7.1

**Comment by jakubmisek** [comment]
Then there is another issue, which will be fixed.

**Comment by jakubmisek** [comment]
the upcoming update won't trigger the fatal error :)

Elementor Pro and WooCommerce were tested as well; of course we can help with that and with the rest

**Comment by jakubmisek** [comment]
fixed in wpdotnet `5.7.8-preview8` https://docs.peachpie.io/scenarios/wordpress/overview/
